lib/charms/operator_libs_linux/v0/juju_systemd_notices.py

```diff
@@ -199,27 +231,31 @@ def subscribe(self) -> None:
                 Restart=always
                 WorkingDirectory={self._charm.framework.charm_dir}
                 Environment="PYTHONPATH={self._charm.framework.charm_dir / "venv"}"
-                ExecStart=/usr/bin/python3 {__file__} {self._charm.unit.name}
+                ExecStart=/usr/bin/python3 {__file__} --unit {self._charm.unit.name} {' '.join(services)}
```

Comment:
Why do we pass `--unit` here now, when we weren't before?

Reply:
Mostly a UX preference for when debugging the notices daemon if there are issues. I try to avoid using args for two or more different objects since it might not always be clear what the order is. E.g. does the unit name come last or first after providing service names: https://clig.dev/#arguments-and-flags

I added the `--unit` flag so it's more clear that it's different from the service names and aliases that are being passed to the notices daemon.